docs/PlatformSpecificInformation.md

```diff
@@ -55,6 +55,43 @@ var styles = StyleSheet.create({
 
 `Platform.OS` will be `ios` when running in iOS and `android` when running in an Android device or simulator.
 
+There is also a `Platform.select` method available, that given an object containing Platform.OS as keys,
+returns the value for the platform you are currently running on.
+
+```javascript
+var { Platform } = React;
+
+var styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    ...Platform.select({
+      ios: {
+        backgroundColor: 'red',
+      },
+      android: {
+        backgroundColor: 'blue',
+      },
+    }),
+  },
+});
+```
+
+This will result in a container having `flex: 1` on both platforms and backgroundColor - red on iOS and blue
+on Android.
+
+Since it accepts `any` value, you can also use it to return platform specific component, like below:
+
+```javascript
+module.exports = class App extends React.Component {
+  render() {
+    return Platform.select({
+      ios: <ViewIOS />,
+      android: <ViewAndroid />,
```

Comment:
This is probably not going to work as you would expect. You're going to require the Android component on iOS and it may very well throw in this environment.

You'll likely want to write this as

```js
var Component = Platform.select({
  ios: () => require('ComponentIOS'),
  android: () => require('ComponentAndroid'),
});
return <Component />;
```